tutor.spec

```diff
@@ -1,7 +1,7 @@
 # -*- mode: python -*-
 import importlib
 import os
-import pkg_resources
+from importlib_metadata
```

Comment:
FTR, this line did cause an issue during the release. It was fixed by https://github.com/overhangio/tutor/commit/b69a8c7adc9adbd34943c820dad12ee2b750c981